tests/legacy/system/test_edb_modeler.py

```diff
@@ -567,3 +567,17 @@ def test_path_center_line(self):
         assert centerline == [[-0.0005, 0.0], [-0.0005, 0.01]]
         edb.modeler.paths[0].center_line = [[0.0, 0.0], [0.0, 5e-3]]
         assert edb.modeler.paths[0].center_line == [[0.0, 0.0], [0.0, 5e-3]]
+
+    def test_polygon_data_refaxtoring_bounding_box(self):
```

Comment:
You can get edbapp from texture

def test_polygon_data_refaxtoring_bounding_box(self, edb_example):
      edbapp = edb_examples.get_si_verse()

Reply:
yes thanks